File: leetcode/rust/find-closest-person.rs
impl Solution {
    pub fn find_closest(x: i32, y: i32, z: i32) -> i32 {
        use std::cmp::Ordering;

        let distance_from_1 = x.abs_diff(z);
        let distance_from_2 = y.abs_diff(z);

        match distance_from_1.cmp(&distance_from_2) {
            Ordering::Less => 1,
            Ordering::Equal => 0,
            Ordering::Greater => 2,
        }
    }
}

File: leetcode/python/find-closest-person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        diff1 = abs(x - z)
        diff2 = abs(y - z)

        if diff1 == diff2:
            return 0

        return 1 if diff1 < diff2 else 2